ui/components/Header.js

```diff
@@ -136,6 +175,15 @@ class Header extends React.Component {
                   <div data-test="notification-button">
                     <MesheryNotification />
                   </div>
+
+                  {/* <Tooltip title="MeshSync Status">
```

Comment:
@Prakhar-Agarwal-byte just a reminder that this is where you'll insert your MeshSync logo logic.

Reply:
@leecalcote Added MeshSync logo in #4785